thavalon-server/src/connections/websockets.rs

```diff
@@ -0,0 +1,56 @@
+use crate::game::PlayerId;
+use futures::channel::mpsc::UnboundedSender;
+use serde;
+use serde_json;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use warp::filters::ws::Message;
+use warp::{Error, Rejection, Reply};
+use warp::ws::{Ws, WebSocket};
+use std::sync::mpsc;
```

Comment:
nit: alphabetical order of imports

Reply:
Deleted this entire file, as it's hot garbage.